react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -64,14 +64,25 @@ const SelectorItem: FC<Props> = ({
     migrationFrom: 'vtex.store-components@3.x',
   })
 
-  const containerClasses = classNames(
-    handles.skuSelectorItem,
-    `${handles.skuSelectorItem}--${slug(variationValue)}`,
-    'relative di pointer flex items-center outline-0 ma2',
-    {
-      [`${handles.skuSelectorItemImage}`]: isImage,
-      'o-20': isImpossible,
-    }
+  const containerClasses = useMemo(
+    () =>
+      classNames(
+        applyModifiers(handles.skuSelectorItem, isSelected ? 'selected' : ''),
+        `${handles.skuSelectorItem}--${slug(variationValue)}`,
```

Comment:
You could also do as follow:

```suggestion
        applyModifiers(handles.skuSelectorItem, [slug(variationValue), isSelected ? 'selected' : '']),
```